sdk/azidentity/cache/README.md

```diff
@@ -0,0 +1,29 @@
+# Azure Identity Cache Module for Go
+
+This module implements a cross-platform persistent token cache for [azidentity][https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity] credentials. See that module's [examples](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity#pkg-examples) for sample code showing how to configure persistent caching for a credential.
```

Comment:
Is the link to `azidentity` supposed to be in parenthesis?

Reply:
Yes! Thanks for spotting this